docs/content-editors.md

```diff
@@ -19,20 +21,21 @@ This project and everyone participating in it, is governed by our [Code of Condu
 
 ## Team Structure
 
-### Project Lead
+This project is the accummulation of the tireless work done by a team of more than 60 volunteers coordinated by Charlie Hoffs
+
+The dev team on this project is led by [Maeva Fincker](https://github.com/mfincker)
+
+
 
-[Kate Hanson](https://github.com/KM-Hanson) is the project lead and was the person who first came up with the idea to build this site.
+## Project Lead
 
-> *TODO: Add more team members here*
+[Joyce Tagal](https://github.com/jayktee) is the project lead for this work and questions or feedback should be directed to her
 
-### Business Owners
 
-If you are a business owner and you would like to add or update your business, just click on our little "Add your business" button at the top right of the site. You will see a dialog pop up which gives you a choice of two forms, one for adding a new business and the second is for you to send us updates for an existing business.
 
-Once you have submitted your form, our content team will do the rest :)
 
 ### Content Ambassadors
 
-Are you able to spare some time to become a volunteer Content Ambassador? We are looking for people to join our enthusiastic team of ambassadors so that we are able to add more businesses and continue to keep the existing businesses up to date.
+Are you able to spare some time to become a volunteer  and help improve / keep the resources or map up to date? We are looking for people to join our enthusiastic team of ambassadors so that we are able to keep this resource up to date through the covid-19 pandemic.  
```

Comment:
```suggestion
Are you able to spare some time to become a volunteer  and help improve / keep the resources or map up to date? We are looking for people to join our enthusiastic team of ambassadors so that we are able to keep this resource up to date through the Covid-19 pandemic.  
```